### ISSUE TITLE
How voting systems influence society

### ISSUE THREAD

**Issue by frivoal** [darobin/politi.es#23]
Choosing between First-past-the-post and ranked-choice voting or other voting systems not only changes who wins the elections, that in turn also changes how political parties organize, who runs for office, how people behave once elected...

The physical organization of the ballot (paper ballots, electronic voting, and many variations) not only changes how easy it is to run the election, but also how easy it is to high-jack the election, or how trust-able the election is.

While these topics seem very technical and boring to many, they can have a profound impact on society, and therefore should not be left to chance or tradition.

There is no single best answer due to the many trade-offs involved, especially when you not only consider the mathematical criteria of voting theory, but also social ones (will most people understand the voting system, trust its outcomes and accept it as fair?) or practical ones (how time consuming is it to run the vote this way?), but that doesn't make the question any less important.

**Comment by darobin** [comment]
I agree that this is an important aspect, but I am unsure how best to approach it. My primary issue with voting models discussions is that they tend to proceed through idealised cases that might not actually bear out in practice.

One example is [liquid democracy](https://en.wikipedia.org/wiki/Delegative_democracy). On paper it seems great to delegate per topic (eg. if it's CSS I delegate to Florian, if it's SVG I delegate to Chris) but that has the knock-on effect that whoever gets to categorise the vote has huge influence (eg. a vote on hit testing of strokes being categorised as SVG or CSS could lead to different results for the same content and the same electors).

Another is more traditional changes to voting, such as [IRV](https://en.wikipedia.org/wiki/Instant-runoff_voting) and [STV](https://en.wikipedia.org/wiki/Single_transferable_vote). They both have sufficient practical deployment that their effects could be studied, but I don't know of such a study. In practice however, given that both are in use in Australia and given [Tony Abbott](https://en.wikipedia.org/wiki/Tony_Abbott) got to be Prime Minister, I am not sure they would save us.

In the spirit of moving this forward, some areas for write-ups:
- What are the empirical conclusions one can draw from alternative electoral systems as deployed today?
- Can we pick one solution for [algorithm district generation](https://en.wikipedia.org/wiki/Gerrymandering#Objective_rules_to_create_districts), make it the norm, and end gerrymandering (which is clear and direct election tampering) worldwide?
- Is there a gold standard for voter identity recognition that minimises both voter suppression and fraud?

Maybe these should be split out as separate issues?

**Comment by astearns** [comment]
Whatever the voting system, my impression is that it's a settled argument that paper ballots are best

**Comment by frivoal** [comment]
@astearns I generally agree, but here's a look into that anyway. Having only one citizenship, I've only voted in one place: France. Here's how our version of paper ballots work: you show up, someone check that you're registered to vote, you pick up an envelope and a bunch of ballots: one separate piece of paper for each candidate, you go into a small curtained booth, put your favorite balot in the envelope and discard the rest, go out of the booth, and put your envelope in a transparent balot box, sign off, and leave.

Upsides:
- This is not time consuming at all, so it doesn't generate long lines
- This is easy to add more booths, so you don't get long limes
- It is easy to set up, so you can have many polling stations
- It is a very simple process, so it is easy to find volunteers to help run it
- It is a very simple process, so it is easy for anyone to make sure it is being run correctly just by being there and watching
- There is close to no risk of ballots being filled incorrectly
- It is simple to count by hand, so no machine is needed
- It is simple to count, so you get results out quickly

Downsides:
- Unmodified, it isn't nearly as practical for elections in which many questions are being asked at the same time
- Unmodified, it doesn't let you do ranked-choice voting

So, given the type of voting we do in France (one question at a time, pick one option), this is close to ideal.

If you do want to do many questions at the time, or if you do want some kind of ranked voting, you'll need to change the formula, generally at the disadvantage of some of the upsides. I think that the fact that evaluating the result of ranked voting by hand is a tedious process for which you tend to want to introduce voting machines (or at least vote counting machines) is a significant downside of it.

**Comment by frivoal** [comment]
> What are the empirical conclusions one can draw from alternative electoral systems as deployed today?

Good question, to which I don't have an answer. I am sure there is academic research about it though. Should we open a dedicated issue or wiki page or page to track that?

>  Can we pick one solution for algorithm district generation, make it the norm, and end gerrymandering (which is clear and direct election tampering) worldwide?

My intuition is not to go for algorithms, but to require that voting districts match administrative districts. It does not make gerrymandering impossible, but it does make it harder to do by adding a lot of inertia to redistricting, and is arguably a lot less mysterious to citizens (and therefore hopefully easier to safeguard) than an algorithmic solution.

>  Is there a gold standard for voter identity recognition that minimizes both voter suppression and fraud?

I doubt it, as it depends on what type of government recognized identity documents exist in the first place, and that question isn't strictly about elections. It seems to me that this topic is much more of an issue in places like the US where it is reasonably likely (and perfectly legal) for a person not to have any form of government recognized ID, and where this can vary by state. As far as I can tell, this isn't much of a problem in France or in Japan, where everybody is supposed to have one (or more) anyway, and the standards for what's a valid one are nation wide.

**Comment by borisschapira** [comment]
A ressource to fuel the discussion http://ncase.me/ballot/

I'm all about the Scoring System.
